README.md

```diff
@@ -0,0 +1,146 @@
+
+## Why We Welcome Contributions
+
+The Open Data Institute is committed to building a strong, fair, and sustainable data economy. We believe that entails involving everybody in the publication and consumption of open data. That’s why we have such a strong focus on data publication in our existing tools - we want **everybody** to be involved in publishing data.
+ 
+The Toolbox grew out of ODI Labs engagement with the open data software ecosystem. Noticing gaps in the types of data tools available Labs developed MVP experimental products intended to plug those gaps. We hope these provocations get everyone thinking. We’d like for the community working in open data software to either fill the identified gaps in other ways, use our work to continue filling those gaps (even locate as yet unidentified gaps!) or contribute to our tools to improve their utility. 
+ 
+We look to OSS success stories like Ruby, Github and Adapt and we want to build an ecosystem of OSS tools that support Open Data - like OKFN’s JKAN. We know we can’t do this all by ourselves, nor would we see this as the best way forward - we aspire to build with developments in the ecosystem. We want to build an OSS community around the Toolbox and hope you can be part of it!
+ 
+We’re focusing on opening the ODI Toolbox to open source contributions to enable users to adapt the tools to their needs by contributing to our code base. We value growing an engaged community to collectively and collaboratively chart a route through our codebase. That way we can identify urgent bug fixes can be applied quickly.
+
+
+## How to Contribute to the ODI Toolbox
+
+### File an Issue for Resolution
+
+If you encounter a problem or have a recommendation for our tools please file an issue in the relevant toolbox repository
+ 
+**Many eyes make all bugs shallow.** 
+We can’t keep track of all the snags that arise across the Toolbox, so if you notice something please let us know.
+ 
+**Suggest a minor enhancement:** 
+We’re not just interested in catching bugs - if you notice something that could be improved about the Tool - whether it’s an interface enhancement or an optimisation suggestion we’re interested to hear from you
+
+### Provide Feedback on the Public Roadmap
+
+The Public Roadmap board lists cards for new features or improvements to these tools, and is intended as an overview of upcoming developments for each tool in the Toolbox.
+The Roadmap tracks future directions for each of the toolbox. You can consider it as the feature backlog for the entire toolbox and per toolbox application. 
+You can also vote and comment on cards to express which ones are most important to you
+
+### Contribute code to our Toolbox
+
+see the next sections for full details
+
+## Contributing Code
+
+If you’re a developer interested in Open Data familiar with any of the following technologies we’d like to interest you in collaborating on our Toolbox
```

Comment:
If you’re a developer interested in Open Data **and** familiar with any of the following technologies**,** we’d like to interest you in collaborating on our Toolbox